tests/foreman/ui/test_subscription.py

```diff
@@ -54,7 +54,7 @@
 
 
 @pytest.fixture(scope='module')
-def content_host_setup():
+def golden_ticket_host_setup():
```

Comment:
Can we mention some details about the golden ticket here in the description?

Reply:
@omkarkhatavkar what kind of details?
There are plenty of materials about "golden ticket" on Mojo and in knowledge base. Also on "Simple Content Access", which is the same idea, but with better name.
Personally, I'm fine with assumption that component owner/person working on these tests knows what "golden ticket" is.